README.md

```diff
@@ -76,6 +76,7 @@ config.providers = [
   {
     provider: :rollbar,
     api_key: ENV['ROLLBAR_KEY'],
+    enabled: Rails.env.production?
```

Comment:
it's obviously all configured by the initializer, but to avoid people just copy/pasting this line, we might want to make it `enabled: !Rails.env.development?` since a few people like to use it in staging